cocos/core/components/model-renderer.ts

```diff
@@ -52,6 +62,44 @@ export class ModelRenderer extends Renderer {
         this._onVisibilityChange(val);
     }
 
+    /**
+     * @zh 组件所属排序层 id，影响组件的渲染排序。
+     * @en The sorting layer id of the component, which affects the rendering order of the component.
+     */
+    // Todo,how to show on inspector
+    get sortingLayerID () {
```

Comment:
Use `sortingLayer` directly in model renderer, and use `sortingLayerIndex` in manager

Reply:
Put public properties on top of all protected properties